cmd/zc_enumerator.go

```diff
@@ -200,6 +204,12 @@ func initResourceTraverser(source string, location common.Location, ctx *context
 			return nil, err
 		}
 
+		if sourceURL.Scheme == "http" {
+			httpsRecommendationOnce.Do(func() {
```

Comment:
Minor: you could define this to reduce duplication in increase brevity:

```
func recommendHttpsIfNecessary(url URL) {
        if url.Scheme.EqualFold("http") {
	    httpsRecommendationOnce.Do(func() { 
				glcm.Info(httpsRecommendedNotice) 
			})
         }
}
```

Then call that everywhere you need it